### DISCUSSION TITLE
After renewing the GitHub Student Developer Pack after a year, will I receive $100 in Azure free credits and Ocean credit again?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65681] [GitHub Education] [answered]
### Hi everyone,

After renewing the GitHub Student Developer Pack following a year, will I still be eligible to receive $100 in Azure free credits and Ocean credit as part of the pack's benefits, or do these specific benefits have renewal criteria or limitations?

**Comment by arafajs** [comment, marked as answer]
@Bakul3014 
The $100 in Azure free credits and Ocean credit are time-based benefits, which means that they are only available for the first year of your GitHub Student Developer Pack. Once your first year is up, you will not be eligible to receive these benefits again, even if you renew your pack.

**Reply by ThreatLEVELred** [reply]
Awesome

**Reply by ThreatLEVELred** [reply]
#[**R```

```**]()

**Comment by ThreatLEVELred** [comment]
NO WAY! BOOOOOOO

**Reply by ThreatLEVELred** [reply]
JK

**Comment by ThreatLEVELred** [comment]
Oh no I have no toilet paper and my battery’s about to run out
/

**Reply by ThreatLEVELred** [reply]
yarn.lock
